symfony/templates/admin/notifications.html.twig

```diff
@@ -0,0 +1,60 @@
+{% extends 'base.html.twig' %}
+{% block title %}Notifications - {% endblock %}
+{% macro displayNotifications(notifications) %}
+    <table class="table table-striped table-hover">
+        <thead><tr><th>Time</th><th>Organization</th><th># Withdrawn Participants</th><th>Notified Emails</th></tr></thead>
+        <tbody>
+        {% for notification in notifications %}
+            <tr>
+                <td data-order="{{ notification.insertTs|date('Y-m-d H:i:s') }}">{{ notification.insertTs|date('n/j/Y g:ia', app.user.timezone) }}</td>
+                <td>{{ notification.hpoId }}</td>
+                <td>{{ notification.count }}</td>
+                <td>{{ notification.email }}</td>
+            </tr>
+        {% endfor %}
+        </tbody>
+    </table>
+{% endmacro %}
+
+{% import _self as notification %}
+
+{% block body %}
+    <ol class="breadcrumb">
+        <li><a href="{{ path('home') }}">Home</a></li>
+        <li><a href="{{ path('admin_home') }}">Admin</a></li>
+        <li class="active">Email Notifications</li>
+    </ol>
+    <div class="page-header">
+        <h2><i class="fa fa-bell" aria-hidden="true"></i> Email Notifications</h2>
+    </div>
+    <div role="tabpanel">
+        <ul class="nav nav-tabs" role="tablist">
+            <li role="presentation" class="active">
+                <a href="#withdrawal" aria-controls="tab" role="tab" data-toggle="tab">Withdrawal Notifications</a>
+            </li>
+            <li role="presentation">
+                <a href="#deactivate" aria-controls="tab" role="tab" data-toggle="tab">Deactivate Notifications</a>
```

Comment:
Change "Deactivate" to "Deactivated". Maybe change the corresponding variables, method names, etc. to match, too.